docs/source/documentation/orchestrator/index.rst

```diff
@@ -2,9 +2,28 @@
 Orchestrator
 ************
 
-The orchestrator has two functions.
-Its first role is to handle the ledger containing all known assets.
-It also dispatches tasks to the relevant :term:`Organizations<Organization>`.
+Performing a Federated Learning experiment implies a lot of different compute tasks: local training, aggregation, testing on different organizations, etc. The role of the orchestrator is to distribute ML tasks among organizations, while ensuring complete traceability of operations.
+
+The orchestrator registers the status of tasks; when a task is done, it evaluates if some tasks in ``Waiting`` are now unblocked, and if it's the case, the status of those tasks is changed to ``To do``. The new status is sent to all the backends, who store the new tasks ``To do`` in the task queue (Celery). Then, the task queue will assign the task to one of the workers (if multiple) and handle retries if needed.
```

Comment:
```suggestion
The orchestrator registers the status of tasks; when a task is done (status ``Done``), it evaluates if some remaining tasks (status ``Waiting``) are now unblocked, and if it's the case, the status of those tasks is changed to ``To do``. The new status is sent to all the backends, who store the new tasks ``To do`` in the task queue (Celery). Then, the task queue will assign the task to one of the workers (if multiple) and handle retries if needed.

In case of failure, it will store fialure reports and  change the status of the faulty task to ``Failed``.
In case of manual cancellation, it will change the status of the  the tasks to `Cancelled` on different backends.
```